doc/manuals/admin/diagnosis.md

```diff
@@ -123,9 +99,69 @@ for the inequity described in [threadpool considerations](perf_tuning.md#thread-
 to hold. Alternatively, the operating system limit could be raised with
 `ulimit -n <new limit>`.
 
+The following script (provides "as is") may help to trace this kind of problems:
+
+```
+echo "$(date +%H:%M:%S) $(/usr/sbin/lsof | grep contextBr |grep IPv4 | wc -l)
+      $(netstat |grep TIME_WAIT |wc -l) $(netstat |grep ESTABLISHED |wc -l)
+      $(tail -200 /var/log/contextBroker/contextBroker.log |grep "Timeout was reached" |wc -l)
+      $(uptime | awk '{print $10" "$11" "$12}' |tr -d ",")
+      $(vmstat |grep -v io |grep -v free)"  &>> /var/log/contextBroker/stats.log
+```
+
+[Top](#top)
+
+### Diagnose thread exahustion problem
```

Comment:
Note that the "big green parts" in the diff are text that has been moved. Thus, typos, etc. in this part is not part of this PR, strictly speaking.

However, if you want to fix the problems you spot, feel free :)